.github/dependabot.yml

```diff
@@ -5,10 +5,6 @@ updates:
     directory: "/"
     schedule:
       interval: weekly
-  - package-ecosystem: pip
-    directory: "docs/"
-    schedule:
-      interval: weekly
```

Comment:
I'm a bit in conflict with this because for the docs code to be healthy and maintained, we would treat it as a regular code part of the project and migrate it to the latest version of its dependencies including the fixes it needs. Making dependabot shut up about it forever goes against keeping the docs code healthy.

Reply:
With this config, dependabot duplicate PR's. Only the first config is needed.

![image](https://github.com/kevin1024/vcrpy/assets/81854/045fcd68-90af-4d03-928f-1181f109d7a5)
![image](https://github.com/kevin1024/vcrpy/assets/81854/495a89ea-00b1-4d97-993a-6d9cf7c4d903)
![image](https://github.com/kevin1024/vcrpy/assets/81854/3fa1d5bf-84ad-4a76-abc6-d86cb68a06d7)

Reply:
> With this config, dependabot duplicate PR's. Only the first config is needed.

@jairhenrique seems like you're right :+1:  #744 and #745 seem to be identical